index.html

```diff
@@ -6,6 +6,12 @@
     <title>If These Lands Could Talk</title>
     <link rel="stylesheet" href="./css/normalize.css" />
     <link rel="stylesheet" href="./css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```

Comment:
Looks great - there looks to be a weird formatting error (maybe via prettier) we can  fix that in a later PR request